packages/wonder-blocks-form/src/components/checkbox-group.tsx

```diff
@@ -129,28 +128,44 @@ const CheckboxGroup = React.forwardRef(function CheckboxGroup(
 
     const allChildren = React.Children.toArray(children).filter(Boolean);
 
+    const legendId = React.useId();
+
     return (
         <StyledFieldset
+            aria-labelledby={legendId}
             data-testid={testId}
             style={[styles.fieldset, style]}
             ref={ref}
         >
-            {label && (
-                <StyledLegend style={styles.legend}>
-                    <LabelMedium>{label}</LabelMedium>
-                </StyledLegend>
-            )}
-            {description && (
-                <LabelSmall style={styles.description}>
-                    {description}
-                </LabelSmall>
-            )}
-            {errorMessage && (
-                <LabelSmall style={styles.error}>{errorMessage}</LabelSmall>
-            )}
-            {(label || description || errorMessage) && (
-                <Strut size={spacing.small_12} />
-            )}
+            <View
```

Comment:
We might want to go about this a different way... To remove `Strut`, I wrapped the affected elements in a `View` and put styling on it instead. But it broke the direct descendant behavior for `fieldset > legend`. I worked around this by explicitly adding an `aria-labelledby` attribute onto the fieldset, so it would have an accessible name exposed. But this might be a case where `Strut` is better suited for the problem.